prisma/migrations/20240408161437_init/migration.sql

```diff
@@ -0,0 +1,58 @@
+-- CreateEnum
+CREATE TYPE "StatusType" AS ENUM ('backlog', 'unstarted', 'started', 'completed', 'cancelled');
+
+-- CreateEnum
+CREATE TYPE "ViewMode" AS ENUM ('board', 'list');
+
+-- CreateEnum
+CREATE TYPE "FilterType" AS ENUM ('person', 'team', 'client', 'all');
+
+-- CreateEnum
+CREATE TYPE "AssigneeType" AS ENUM ('iu', 'client', 'company');
+
+-- CreateTable
+CREATE TABLE "StatusSettings" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "type" "StatusType" NOT NULL,
+    "name" TEXT NOT NULL,
+    "key" TEXT NOT NULL,
+    "color" TEXT,
+
+    CONSTRAINT "StatusSettings_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Tasks" (
```

Comment:
@rrojan are we going to  use `PascalCase` for table name? CC: @sazanrjb .

if yes, then it should we make the column names `PascalCase` as well?

Reply:
@saugatmaharjan  The table names were suggested by the Copilot Team on their PRD [here](https://copilotplatforms.notion.site/Tasks-App-4e19e810bf0c4c8b96e9d583dc42c400#f10a8acfb3144225b5cbd93f7fa73a3c)

Reply:
@rrojan the names suggested in the PRD are not table names, those were the resource names. You can use a different pattern for naming tables